scripts/upgrade

```diff
@@ -39,20 +30,6 @@ auth_method=$(ynh_app_setting_get --app=$app --key=auth_method)
 
 upgrade_type=$(ynh_check_app_version_changed)
 
-#=================================================
-# BACKUP BEFORE UPGRADE THEN ACTIVE TRAP
-#=================================================
-ynh_script_progression --message="Backing up the app before upgrading (may take a while)..." --weight=1
-
-# Backup the current version of the app
-ynh_backup_before_upgrade
-ynh_clean_setup () {
-	# Restore it if the upgrade fails
-	ynh_restore_upgradebackup
-}
-# Exit if an error occurs during the execution of the script
-ynh_abort_if_errors
-
 #=================================================
 # ENSURE DOWNWARD COMPATIBILITY
```

Comment:
below you could in fact "simplify" stuff by using

```
if [ -z "${fpm_footprint:-}" ];
```

(instead of `if [ -z "$fpm_footprint" ]`)

such that you don't have to load `fpm_footprint=$(ynh_app_setting_get --app=$app --key=fpm_footprint)` earlier:
- if the setting exists, it will be loaded in the env
- othewise, `${fpm_footprint:-}` will return empty string without triggering the "unboud variable" error, and then the content of the `if` block will initialize it

same for fpm_free_footprint, fpm_usage, auth_method